src/index.html

```diff
@@ -1,13 +1,360 @@
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="events-lection">
+        <img
+          src="./images/events-card-3.png"
+          alt="events lection"
+          class="events-lection__img"
+        >
+        <div class="events-lection__box">
+          <div class="events-lection__head">
+            <h3 class="events-lection__subtitle">ЛЕКЦІЯ</h3>
+            <p class="events-lection__date">26/08/2019 - 11:00</p>
+          </div>
+          <h2 class="events-lection__title">Від класицизму до романтизму</h2>
+        </div>
+      </div>
+    </section>
+    <!-- #Events -->
+
+    <!-- Gallery -->
+    <div class="gallery" id="gallery">
+      <div class="container">
+        <div class="gallery__slides">
+          <div class="gallery__slide gallery__slide-1">
+            <img
+            src="./images/gallery-1.png"
+            alt="slide-1"
+            class="gallery__img"
+            >
+          </div>
+
+          <div class="gallery__slide gallery__slide-2">
+            <img
+            src="./images/gallery-2.png"
+            alt="slide-2"
+            class="gallery__img gallery__img-2"
+            >
+          </div>
+
+          <div class="gallery__slide gallery__slide-3">
+            <img
+            src="./images/gallery-3.png"
+            alt="slide-3"
+            class="gallery__img gallery__img-3"
+            >
+          </div>
+
+          <div class="gallery__slide gallery__slide-4">
+            <img
+            src="./images/gallery-4.png"
+            alt="slide-4"
+            class="gallery__img gallery__img-4"
+            >
+          </div>
+
+        </div>
+        <div class="gallery__nav">
+          <div class="gallery__nav-btn"></div>
+          <div class="gallery__nav-btn"></div>
+          <div class="gallery__nav-btn"></div>
+          <div class="gallery__nav-btn"></div>
+        </div>
+      </div>
+    </div>
+    <!-- #Gallery -->
+
+    <!-- Subscribe -->
+    <section class="subscribe" id="about">
+      <div class="container">
+        <div class="subscribe__wrapper">
+          <h2 class="subscribe__title">Підписка</h2>
+          <p class="subscribe__text">Бути в курсі всіх актуальних подій та останніх новин.</p>
+          <form
+            action="#"
+            class="subscribe__form"
+            onsubmit="this.reset(); return false;"
+          >
+            <label for="input" class="form__label"></label>
+            <input
+              type="email"
+              class="subscribe__input"
+              id="input"
+              placeholder="email"
+            >
+            <button class="subscribe__btn">></button>
+          </form>
+        </div>
+      </div>
+    </section>
+    <!-- #Subscribe -->
+
+    <!-- Footer -->
+    <section class="footer">
```

Comment:
Use a semantic tag for footer
```suggestion
    <footer class="footer">
```